internal/utils.go

```diff
@@ -46,6 +46,9 @@ func FormatCacheSize(size_in_bytes float64) string {
 func PrettyPrintCacheList(caches []types.ActionsCache) {
 	terminal := ghTerm.FromEnv()
 	w, _, _ := terminal.Size()
+	if w == 0 {
```

Comment:
What is the case when this is `0`? is it only during test as that runs non-interactively? If yes, might be good to add this as comment here.